client/blocks/stats-sparkline/index.jsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { isJetpackSite, getSiteDomain } from 'state/sites/selectors';
+
+const StatsSparkline = ( { isJetpack, siteDomain, className, siteId } ) => {
+	if ( ! siteId || ! siteDomain || isJetpack ) {
+		return null;
+	}
+
+	return (
+		<img
+			className={ className }
+			src={ `https://${ siteDomain }/wp-includes/charts/admin-bar-hours-scale-2x.php?masterbar=1&s=${ siteId }` } />
+	);
+};
+
+StatsSparkline.propTypes = {
+	className: PropTypes.string,
+};
+
+export default connect(
+	( state, ownProps ) => {
+		const { siteId } = ownProps;
```

Comment:
If we rely on a siteId prop and use some site selectors, we should probably add a `QuerySite`, don't you think?
An alternative would be to rely on the selectedSiteId